packages/react/src/Hero/Hero.stories.tsx

```diff
@@ -14,125 +15,105 @@ export default {
   }
 } as ComponentMeta<typeof Hero>
 
-const Template: ComponentStory<typeof Hero> = args => <Hero {...args} />
+export const Default: ComponentStory<typeof Hero> = _args => (
+  <Hero>
+    <Hero.Heading>This is my super sweet hero heading</Hero.Heading>
+    <Hero.Description>
+      Lorem ipsum dolor sit amet, consectetur adipiscing elit. In sapien sit ullamcorper id. Aliquam luctus sed turpis
+      felis nam pulvinar risus elementum.
+    </Hero.Description>
+    <Hero.PrimaryAction href="#">Primary action</Hero.PrimaryAction>
+  </Hero>
+)
 
-export const Default = Template.bind({})
-Default.args = {
-  heading: 'This is my super sweet hero heading',
-  description:
-    'Lorem ipsum dolor sit amet, consectetur adipiscing elit. In sapien sit ullamcorper id. Aliquam luctus sed turpis felis nam pulvinar risus elementum.',
-  primaryAction: {
-    text: 'Primary action',
-    href: '#'
-  }
-}
+export const Centered: ComponentStory<typeof Hero> = _args => (
```

Comment:
Could we move all stories besides the default and playground into a new file called `Hero.features.stories.tsx` please? This is happening for all new components per an ADR. [Here's an example for ActionMenu](https://primer-cfabcb7d9c-26139705.drafts.github.io/storybook/?path=/story/components-actionmenu--default).